drivers/wifi/esp_at/Kconfig.esp_at

```diff
@@ -1,110 +1,114 @@
 # Copyright (c) 2019 Tobias Svehagen
 # SPDX-License-Identifier: Apache-2.0
 
-menuconfig WIFI_ESP
-	bool "Espressif ESP8266 and ESP32 support"
+menuconfig WIFI_ESP_AT
+	bool "Espressif AT Command support"
```

Comment:
Is this clear enough description? I'm not sure. Possible ideas: `Espressif devices via AT commands`, `Espressif devices via serial`, `Espressif ESP8266/ESP32 via AT commands`, etc.

Reply:
The idea behind `Espressif AT Command ...` was that this WiFi driver implements/utilizes AT commands defined by Espressif. So more like "Espressif's AT command interface". With proposed `Espressif devices via ...` it suggests that only Espressif devices could be used. This is true for now, but in theory someone could implement exactly the same functionality on other vendor's WiFi chip, but just conforming to the same AT commands as [esp-at](https://github.com/espressif/esp-at) repository does. In the end, I don't have strong preference here.